README.md

```diff
@@ -1,10 +1,13 @@
+
```

Comment:
Nit: there's a spurious newline here at the beginning

Reply:
I removed the line, good catch.